src/test/java/picard/util/FifoBufferTest.java

```diff
@@ -57,8 +57,8 @@ public void test(final double megabytes) throws IOException {
         }
 
         // Run the input file through the FifoBuffer and back into another file
-        final Md5CalculatingInputStream in   = new Md5CalculatingInputStream(new FileInputStream(inputFile), null);
-        final Md5CalculatingOutputStream out = new Md5CalculatingOutputStream(new FileOutputStream("/dev/null"), null);
+        final Md5CalculatingInputStream in   = new Md5CalculatingInputStream(new FileInputStream(inputFile), (File)null);
+        final Md5CalculatingOutputStream out = new Md5CalculatingOutputStream(new FileOutputStream("/dev/null"), (File)null);
```

Comment:
It would be nice to update this code at some point. E.g. use `Path` not `File` and use try/catch to open the files to avoid the need to `close()`.